addons/forward/models.py

```diff
@@ -44,7 +44,8 @@ def save(self, request=None, *args, **kwargs):
             else:
                 user = request.user
 
-            self.owner.check_spam(user, {'addons_forward_node_settings__url'}, get_headers_from_request(request))
+            if self.owner.get_addon('forward'):
```

Comment:
This conditional seems superfluous -- `self.owner.get_addon('forward') == self`, no?